scenario/state.py

```diff
@@ -609,6 +610,80 @@ class Mount(_DCBase):
     src: Union[str, Path]
 
 
+def _now_utc():
+    return datetime.datetime.now(tz=datetime.timezone.utc)
+
+
+_next_notice_id_counter = 1
+
+
+def next_notice_id(update=True):
```

Comment:
Does this need to be `next_pebble_notice_id`? It would be more consistent and avoid problems if there's some other kind of `[X]Notice` type in the future, but it seems pretty wordy.